packages/web-components/src/components/quote/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import '../quote';
+
+# Quote
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components/examples/codesandbox/components/quote)
+> example implementation.
+
+[![Edit @carbon/ibmdotcom-web-components](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components/examples/codesandbox/components/quote)
+
+## Getting started
+
+Here's a quick example to get you started.
+
+### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/quote/quote';
+import '@carbon/ibmdotcom-web-components/es/components/link-with-icon/link-with-icon';
+import 'carbon-web-components/es/icons/arrow--right/20.js';
+```
+
+or with [JSPM](https://jspm.org)
+
+```html
+<script type="module">
+  import 'https://jspm.dev/@carbon/ibmdotcom-web-components@canary/es/components/quote/quote.js';
+  import 'https://jspm.dev/@carbon/ibmdotcom-web-components@canary/es/components/link-with-icon/link-with-icon.js';
+  import 'carbon-web-components/es/icons/arrow--right/20.js';
+</script>
+```
+
+### HTML
+
+```html
+<dds-quote inverse="false" markType="${quoteMark}">
```

Comment:
Boolean attribute should be "attribute exists or not" thing unless it's `aria-*`. Also, we use dash case for attribute names (whereas we use came case for property names). Also I think we don't want to put `lit-html` example here:

```suggestion
<dds-quote inverse mark-type="singleCurved">
```